code/ui/components/src/tooltip/ListItem.tsx

```diff
@@ -126,7 +126,7 @@ const Item = styled.a<ItemProps>(
   ({ theme }) => ({
     fontSize: theme.typography.size.s1,
     transition: 'all 150ms ease-out',
-    color: transparentize(0.5, theme.color.defaultText),
+    color: theme.color.defaultText,
```

Comment:
Thanks for this PR! Can you please change this to `theme.color.dark`. It should resolve the issue and be the color that was originally proposed with the menu update.

Reply:
done!